documentation/breaking-changes/upcoming-breaking-changes.md

```diff
@@ -2,6 +2,11 @@
 
 ## Az.Accounts
 
+### `Resolve-AzError`
```

Comment:
Should this file be generated or manually updated? Remember it will be generated all at once before the release.